Lib/pickle.py

```diff
@@ -393,13 +421,21 @@ def __init__(self, file, protocol=None, *, fix_imports=True):
         will try to map the new Python 3 names to the old module names
         used in Python 2, so that the pickle data stream is readable
         with Python 2.
+
+        If *buffer_callback* is None (the default), buffer views are
+        serialized into *file* as part of the pickle stream.  It is
+        an error if *buffer_callback* is not None and *protocol* is
+        None or smaller than 5.
```

Comment:
Maybe also document the non-None case here?

Reply:
Will do, thanks.